SEED-platform/seed
# Add derived data

To test:
1. Create derived columns
2.  Look at those columns, they should be empty!
3. call the action "Update derived data on them". They will turn yellow, and update.
4. edit a field that'll change the derived data and rerun step 3. 

TO COME: calling the action automatically.
